omen2/types.py

```diff
@@ -34,9 +34,11 @@ def bool_type(arg):
 
 def string_type(arg):
     """Convert sql string to str, this function must have the __name__ 'str'"""
-    assert arg[0] in ("'", '"')
-    # return with quotes
-    return arg
+    if arg[0] in ("'", '"'):
+        return arg
+    else:  # pragma: no cover
```

Comment:
seems like a bug paver

Reply:
sqlglot doesn't normalize defaults in any way.   so we're kinda left to our own devices on that

Reply:
probably going to save that for after mssql support, which i think is more important

Reply:
i'll try to write a test for this

Reply:
same issue as above, in sqlite if you have a default (4) for a text column, you can get this.   again, think its ok to remove

Reply:
okay understood